### DISCUSSION TITLE
Axis Cameras on IPv6 via VAPIX

### DISCUSSION THREAD

**Discussion by harshvir** [AxisCommunications/discussions#199] [VAPIX] [answered]
We are currently using IPv4 network and camera configuration to stream from Axis Cameras, and we are planning to move to IPv6 network.
I have checked that camera configuration supports IPv6.
Once enabled then camera will be working on IPv6 network.
Do I need to change my implementation the way we call VAPIX API using http commands, will I need to change my code or just updating the IP will be enough?

Additionally, Can someone point me to an example of using VAPIX API on IPv6 camera configuration?

**Comment by ghost** [comment, marked as answer]
I am unsure if I completely understand your question, but using postman is what I would use for the URL for my camera if I were using IPv6.

http://[fe80::baa4:4fff:fe49:e268]/axis-cgi/param.cgi?action=list

In all the VAPIX examples, they leave it up to the user to either use IPv6 or IPv4 address.  To getNetworkInfo go use this URL and search for Read network configuration   https://www.axis.com/vapix-library/subjects/t10175981/section/t10136094/display?section=t10136094-t10136371

**Reply by harshvir** [reply]
Thanks for the information. My question was more on the lines if the same postman commands that we have for IPv4 will continue to work with IPv6 enabled.
We use VAPIX commands for multiple operations starting with creating/enabling/disabling privacy masks, getting audio notifications. We also rely on getting notification from Camera via http commands for motion detection and I/O modules.

**Reply by ghost** [reply]
Yes, the command should be the same; the only difference is the server address, IPv4, or IPv6 address.
